drift/test/generated/todos.g.dart

```diff
@@ -760,25 +816,66 @@ class $UsersTable extends Users with TableInfo<$UsersTable, User> {
   late final GeneratedColumn<int> id = GeneratedColumn<int>(
       'id', aliasedName, false,
       type: DriftSqlType.int,
-      requiredDuringInsert: false,
-      defaultConstraints: 'PRIMARY KEY AUTOINCREMENT');
+      requiredDuringInsert: false, defaultConstraints: (context) {
+    const dialectConstraints = {
+      SqlDialect.sqlite: 'PRIMARY KEY AUTOINCREMENT',
+      SqlDialect.mysql: 'PRIMARY KEY AUTOINCREMENT',
+      SqlDialect.postgres: 'PRIMARY KEY AUTOINCREMENT',
+    };
+
+    final constraints = dialectConstraints[context.dialect]!;
+    if (constraints.isEmpty) {
+      return;
+    }
+
+    context.buffer
+      ..write(' ')
+      ..write(constraints);
+  }, hasAutoIncrement: true);
   final VerificationMeta _nameMeta = const VerificationMeta('name');
   @override
   late final GeneratedColumn<String> name = GeneratedColumn<String>(
       'name', aliasedName, false,
       additionalChecks:
           GeneratedColumn.checkTextLength(minTextLength: 6, maxTextLength: 32),
       type: DriftSqlType.string,
-      requiredDuringInsert: true,
-      defaultConstraints: 'UNIQUE');
+      requiredDuringInsert: true, defaultConstraints: (context) {
+    const dialectConstraints = {
+      SqlDialect.sqlite: 'UNIQUE',
+      SqlDialect.mysql: 'UNIQUE',
+      SqlDialect.postgres: 'UNIQUE',
+    };
+
+    final constraints = dialectConstraints[context.dialect]!;
+    if (constraints.isEmpty) {
+      return;
+    }
+
+    context.buffer
+      ..write(' ')
+      ..write(constraints);
+  });
   final VerificationMeta _isAwesomeMeta = const VerificationMeta('isAwesome');
   @override
   late final GeneratedColumn<bool> isAwesome = GeneratedColumn<bool>(
       'is_awesome', aliasedName, false,
       type: DriftSqlType.bool,
-      requiredDuringInsert: false,
-      defaultConstraints: 'CHECK ("is_awesome" IN (0, 1))',
-      defaultValue: const Constant(true));
+      requiredDuringInsert: false, defaultConstraints: (context) {
+    const dialectConstraints = {
+      SqlDialect.sqlite: 'CHECK ("is_awesome" IN (0, 1))',
+      SqlDialect.mysql: 'CHECK ("is_awesome" IN (0, 1))',
+      SqlDialect.postgres: 'CHECK ("is_awesome" IN (0, 1))',
```

Comment:
I assumed we don't need this check anymore in postgres? Am I overlooking something?

Reply:
You're right, I forgot to re-run `melos run build` after adding the code to handle this.